packages/gatsby-theme-carbon/src/components/LeftNav/ResourceLinks.js

```diff
@@ -7,18 +7,23 @@ const LeftNavResourceLinks = ({ links }) =>
   links ? (
     <>
       <hr className="bx--side-nav__divider" />
-      {links.map((link, i) => (
-        <SideNavLink
-          key={i}
-          style={{ marginTop: i === 0 ? '1rem' : 0 }}
-          icon={<LaunchIcon />}
-          href={link.href}
-          className="bx--side-nav--website-link"
-          element={!link.href.includes('http') ? Link : 'a'}
-        >
-          {link.title}
-        </SideNavLink>
-      ))}
+      {links.map((link, i) => {
+        const linkProps = !link.href.includes('http')
+          ? { element: Link }
+          : { element: 'a', target: '_blank', rel: 'noopener noreferral' }
+        return (
+          <StyledLink
```

Comment:
Did you mean to change this to `<StyledLink>` ? getting a build error
```
15 |           <StyledLink
     |           ^
  16 |             key={i}
  17 |             style={{ marginTop: i === 0 ? '1rem' : 0 }}
  18 |             icon={<LaunchIcon />}


  WebpackError: ReferenceError: StyledLink is not defined
  
  - ResourceLinks.js:15 
```

Reply:
oof, I was wrapping it to change some styling. Changed it back